proposals/top-level-statements.md

```diff
@@ -81,15 +81,15 @@ static class Program
 }
 ```
 
-Note that the names "Program" and "Main" are used only for illustrations purposes, actual names used by
-compiler are implementation dependent and neither the type, nor the method can be referenced by name from
+Note that the names "Program" and "Main" are used only for illustration purposes, actual names used by the
+compiler are implementation-dependent and neither the type, nor the method can be referenced by name from
 source code.
 
 The method is designated as the entry point of the program. Explicitly declared methods that by convention 
-could be considered as an entry point candidates are ignored. A warning is reported when that happens. It is
-an error to specify `-main:<type>` compiler switch when there are top-level statements.
+could be considered as entry point candidates are ignored. A warning is reported when that happens. It is
+an error to specify the `-main:<type>` compiler switch when there are top-level statements.
 
-The entry point method always has one formal parameter, ```string[] args```. The execution environment creates and passes a ```string[]``` argument containing the command-line arguments that were specified when the application was started. The ```string[]``` argument is never null, but it may have a length of zero if no command-line arguments were specified. The ‘args’ parameter is in scope  within top-level statements and is not in scope outside of them. Regular name conflict/shadowing rules apply.
+The entry point method always has one formal parameter; ```string[] args```. The execution environment creates and passes a ```string[]``` argument containing the command-line arguments that were specified when the application was started. The ```string[]``` argument is never null, but it may have a length of zero if no command-line arguments were specified. The ‘args’ parameter is in scope  within top-level statements and is not in scope outside of them. Regular name conflict/shadowing rules apply.
```

Comment:
All of these changes to `;` are incorrect. The sentence is not ending, it is showing an example. Same with the example changes below.

Reply:
The use of `;` in the context of line 92 is correct, `one formal parameter` refers to, and directly precedes, `string[] args`, and, thus, a semicolon should be used to punctuate this.

Reply:
It is not correct. Semicolons are used to break up 2 sentences; each sentence should be complete on its own. ```string[] args``` is not a complete sentence. A `:` would be acceptable, as is a `,` as written here. Please revert all conversions to `;`.